switches/Ajazz/Dicefruitkiwi.json

```diff
@@ -0,0 +1,12 @@
+{
+     "Brand": "Ajazz",
+     "Switch": "Diced Fruit Kiwi",
+     "Feel": "Tactile",
+     "ActivationForce": "-",
+     "BottomOutForce": "50",
+     "ActivationPoint": "1.9",
+     "TravelDistance": "3.3",
+     "RegEx": "(Diced | kiwi)",
```

Comment:
Consider a more flexible regex as the uppercase "D" Diced will not be matched if someone types `/switch dicedkiwi`.

Reply:
Agreed, be sure to take a look at some of our other switches, (going to be updating the bot so it removes uppercase letters and spaces from the user's input so that should help too)

Reply:
@jathanasiou bot has been updated